services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/ThrottlingReroutesSpec.groovy

```diff
@@ -0,0 +1,272 @@
+package org.openkilda.functionaltests.spec
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.FlowHelper
+import org.openkilda.functionaltests.helpers.PathHelper
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.info.event.IslChangeType
+import org.openkilda.messaging.info.event.PathNode
+import org.openkilda.messaging.model.SwitchId
+import org.openkilda.messaging.payload.flow.FlowState
+import org.openkilda.northbound.dto.switches.PortDto
+import org.openkilda.testing.model.topology.TopologyDefinition
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+import org.openkilda.testing.service.database.Database
+import org.openkilda.testing.service.northbound.NorthboundService
+import org.openkilda.testing.service.topology.TopologyEngineService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import spock.lang.Narrative
+
+import java.util.concurrent.TimeUnit
+
+import static org.junit.Assume.assumeTrue
```

Comment:
Usually, static imports are located right after `package some_package_name`

Reply:
Our checkstyle misses this for Groovy. Will fix here and add the correct order to checkstyle.xml as a separate patch

Reply:
Sure.